### PR TITLE
refactor(js): rename clientId to appId in js SDKs

### PR DESCRIPTION
**Pull request by demonzoo**
## Summary

Rename `clientId` to `appId` in JS SDKs

## Linear Issue Reference

[LOG-2252](https://linear.app/silverhand/issue/LOG-2252/rename-client-id-to-app-id-in-sdks)

## Testing

- [x] Passed all unit tests
- [x] Tested browser sample and it worked
- [x] Tested react sample and it also worked

### CODE REVIEW

File: packages/browser-sample/src/index.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-import { endpoint, clientId } from './consts.js';
+import { endpoint, appId } from './consts.js';
 import Callback from './pages/Callback.js';
 import Home from './pages/Home.js';
 
 import './index.scss';
 
-const logtoClient = new LogtoClient({ endpoint, clientId });
+const logtoClient = new LogtoClient({ endpoint, appId });
+console.log('logto client:', logtoClient);
```
Comment (wangsijie) [inline packages/browser-sample/src/index.js:10]: Is this on purpose?

**Review by wangsijie — APPROVED** [review]
Overall LGTM.

File: packages/browser-sample/src/index.js (hunk 1 above)
Comment (demonzoo) [reply packages/browser-sample/src/index.js:10]: Oops, should remove it. Thanks

File: packages/js/src/core/fetch-token.test.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ describe('fetch access token by providing valid refresh token', () => {
 
     const tokenResponse = await fetchTokenByRefreshToken(
       {
-        clientId: 'client_id_value',
```
Comment (gao-sun) [inline packages/js/src/core/fetch-token.test.ts:64]: we can keep core client id since it directly contacts oidc?

Comment (demonzoo) [reply packages/js/src/core/fetch-token.test.ts]: OK, reverted all the changes made in js core, keeping only those in `browser` and above layers

**Pull request merged**